CondCore/SiPixelPlugins/plugins/BuildFile.xml

```diff
@@ -1,58 +1,49 @@
+<use name="CondCore/SiPixelPlugins"/>
+
 <library file="SiPixelQuality_PayloadInspector.cc" name="SiPixelQuality_PayloadInspector">
   <use name="CondCore/Utilities"/>
   <use name="CondCore/CondDB"/>
-  <use name="CondFormats/Common"/>
   <use name="CalibTracker/StandaloneTrackerTopology"/>
-  <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
 
 <library file="SiPixelLorentzAngle_PayloadInspector.cc" name="SiPixelLorentzAngle_PayloadInspector">
   <use name="CondCore/Utilities"/>
   <use name="CondCore/CondDB"/>
-  <use name="CondFormats/Common"/>
   <use name="CalibTracker/StandaloneTrackerTopology"/>
-  <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
 
 <library file="SiPixelGainCalibrationOffline_PayloadInspector.cc" name="SiPixelGainCalibrationOffline_PayloadInspector">
   <use name="CondCore/Utilities"/>
   <use name="CondCore/CondDB"/>
-  <use name="CondFormats/Common"/>
   <use name="CalibTracker/StandaloneTrackerTopology"/>
-  <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
 
 <library file="SiPixelGainCalibrationForHLT_PayloadInspector.cc" name="SiPixelGainCalibrationForHLT_PayloadInspector">
   <use name="CondCore/Utilities"/>
   <use name="CondCore/CondDB"/>
-  <use name="CondFormats/Common"/>
   <use name="CalibTracker/StandaloneTrackerTopology"/>
-  <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
 
 <library file="SiPixelTemplateDBObject_PayloadInspector.cc" name="SiPixelTemplateDBObject_PayloadInspector">
   <use name="CondCore/Utilities"/>
   <use name="CondCore/CondDB"/>
-  <use name="CondFormats/Common"/>
   <use name="CondFormats/SiPixelTransient"/>
   <use name="CalibTracker/StandaloneTrackerTopology"/>
-  <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
 
 <library file="SiPixelVCal_PayloadInspector.cc" name="SiPixelVCal_PayloadInspector">
   <use name="CondCore/Utilities"/>
   <use name="CondCore/CondDB"/>
-  <use name="CondFormats/Common"/>
   <use name="CalibTracker/StandaloneTrackerTopology"/>
-  <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
 
+<<<<<<< HEAD
```

Comment:
this cannot be an intended change.

Reply:
Right, it's not! Thanks

Reply:
Didn't catch it because it compiled anyway...